ClientApp/viking-vault/src/components/styles.css

```diff
@@ -131,23 +129,30 @@
 }
 
 .side-menu-true {
```

Comment:
Please try to find a better way to express a boolean state that an HTML component can have rather than having "class-" + "true"/"false", this is applicable all throughout the application, not just here